src/components/usa-button-group/button-group.stories.js

```diff
@@ -0,0 +1,21 @@
+import "../../stylesheets/uswds.scss";
+import Component from "./usa-button-group.twig";
+import Data from "./usa-button-group.json";
+import SegmentedData from "./usa-button-group~segmented.json";
```

Comment:
When there are multiple JSON files, can we standardize the file structure inside a content folder like the button and search components?

Reply:
Going to take care of this in Phase 2 #4466